interpreter/cling/tools/plugins/clad/CMakeLists.txt

```diff
@@ -20,87 +20,51 @@ if("${CMAKE_GENERATOR}" STREQUAL "Ninja")
-    # support building prebuilt clang and adding clangBasic breaks this case.
-    # Delegate the dependency resolution to the clingInterpreter target (which
-    # will always depend on clangBasic).
-    DEPENDS clingInterpreter
-  )
-else()
-  set(_clad_build_type ${CMAKE_CFG_INTDIR})
-  if(APPLE)
-    set(_clad_extra_cmake_args
-      -DCMAKE_OSX_SYSROOT=${CMAKE_OSX_SYSROOT}
-    )
-  endif()
+if(APPLE)
+  set(_clad_extra_cmake_args -DCMAKE_OSX_SYSROOT=${CMAKE_OSX_SYSROOT})
+endif()
 
-  ExternalProject_Add(
-    clad
-    GIT_REPOSITORY https://github.com/vgvassilev/clad.git
-    GIT_TAG v0.7
-    UPDATE_COMMAND ""
-    CMAKE_ARGS -G ${CMAKE_GENERATOR}
-               -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-               -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
-               -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
-               -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
-               -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
-               -DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}
-               -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
-               -DLLVM_DIR=${LLVM_DIR}
-               -DClang_DIR=${Clang_DIR}
-               -DCLANG_INCLUDE_DIRS=${CLANG_INCLUDE_DIRS}
-               -DCLAD_BUILD_STATIC_ONLY=ON
-               ${_clad_extra_cmake_args}
-    INSTALL_COMMAND env DESTDIR=${CMAKE_BINARY_DIR}/plugins ${CMAKE_COMMAND} --build . --target install
-               COMMAND  ${CMAKE_COMMAND} -E copy_directory ${CMAKE_BINARY_DIR}/plugins/${CMAKE_INSTALL_PREFIX}/ ${clad_install_dir}/plugins
-    BUILD_BYPRODUCTS ${CLAD_BYPRODUCTS}
-    # Wrap download, configure and build steps in a script to log output
-    LOG_DOWNLOAD ON
-    LOG_CONFIGURE ON
-    LOG_BUILD ON
-    LOG_INSTALL ON
-    # We need the target clangBasic to be built before building clad. However, we
-    # support building prebuilt clang and adding clangBasic breaks this case.
-    # Delegate the dependency resolution to the clingInterpreter target (which
-    # will always depend on clangBasic).
-    DEPENDS clingInterpreter
+ExternalProject_Add(
+  clad
+  GIT_REPOSITORY https://github.com/vgvassilev/clad.git
+  GIT_TAG v0.7
+  UPDATE_COMMAND ""
+  CMAKE_ARGS -G ${CMAKE_GENERATOR}
+             -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+             -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+             -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
+             -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
+             -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
+             -DCMAKE_CXX_STANDARD=${CMAKE_CXX_STANDARD}
+             -DCMAKE_INSTALL_PREFIX=${clad_install_dir}/plugins
+             -DLLVM_DIR=${LLVM_DIR}
+             -DClang_DIR=${Clang_DIR}
+             -DCLANG_INCLUDE_DIRS=${CLANG_INCLUDE_DIRS}
+             -DCLAD_BUILD_STATIC_ONLY=ON
+             ${_clad_extra_cmake_args}
+  BUILD_COMMAND ${CMAKE_COMMAND} --build . --config ${_clad_build_type}
+  INSTALL_COMMAND ${CMAKE_COMMAND} --build . --config ${_clad_build_type} --target install
```

Comment:
@oshadura, does fixing the `CMAKE_INSTALL_PREFIX` resolve the issue you fixed with `DESTDIR`?

Reply:
@oshadura I tested:
```
~/workspace/builds/root_inst$ root.exe -l -b
root [0] #include <Math/CladDerivator.h>
root [1] double f (double x) { return x*x ; }
root [2] clad::differentiate(f, 0)
(clad::CladFunction<false, double, double>) @0x7fe5c1cb26c0
root [3] .q

```

That should confirm the original issue seems to be fixed.